src/views/integrations/event-tickets/emails/template-parts/body/event/description.php

```diff
@@ -27,7 +27,7 @@
 
 ?>
 <tr>
-	<td style="padding:0;" class="tec-tickets__email-table-content-event-description-container">
+	<td style="padding-bottom:20px;" class="tec-tickets__email-table-content-event-description-container">
```

Comment:
I think for consistency, if we're moving the post title to ET, we should also do it for the excerpt.

Reply:
The styling should be moved into the `styles.php` file located in the `template-parts/header/head/` folder.